src/lockscreen.js

```diff
@@ -86,12 +110,34 @@ var Lockscreen = GObject.registerClass({
         if (!this._key)
             return;
         if (this._manager.hasKey(this._key)) {
+            this._style.remove_class('locked');
```

Comment:
What needs to happen here is that the lock-screen should update the image to another image that tells the user that "know it can be opened".

Reply:
To be more clear, we are missing one asset per lock-screen now.